cluster/gce/win1803/k8s-node-setup.psm1

```diff
@@ -261,15 +261,17 @@ function DownloadAndInstall-KubernetesBinaries {
 
   # Disable progress bar to dramatically increase download speed.
   $ProgressPreference = 'SilentlyContinue'
-  Invoke-WebRequest `
-    https://storage.googleapis.com/kubernetes-release/release/${k8sVersion}/bin/windows/amd64/kubectl.exe `
-    -OutFile ${env:NODE_DIR}\kubectl.exe
-  Invoke-WebRequest `
-    https://storage.googleapis.com/kubernetes-release/release/${k8sVersion}/bin/windows/amd64/kubelet.exe `
-    -OutFile ${env:NODE_DIR}\kubelet.exe
-  Invoke-WebRequest `
-    https://storage.googleapis.com/kubernetes-release/release/${k8sVersion}/bin/windows/amd64/kube-proxy.exe `
-    -OutFile ${env:NODE_DIR}\kube-proxy.exe
+
```

Comment:
This is some-what hacky, but I've tested and it worked.
We need to clean up more code around the directoris.
@pjh could you take a look?

Reply:
I made some comments about cleaning up the directories, as you noted, but yeah this approach seems fine to me.